backend/compact-connect/lambdas/common-python/cc_common/data_model/client.py

```diff
@@ -196,6 +196,7 @@ def create_privilege(self, *, compact: str, jurisdiction: str, provider_id: str)
                                     'status': provider_data['status'],
                                     'dateOfExpiration': provider_data['dateOfExpiration'],
                                     'dateOfIssuance': now.date(),
+                                    'dateOfRenewal': now.date(),
```

Comment:
Accessibility: Duplicate id attribute value "compact" found on the web page.

<img width="478" alt="image" src="https://github.com/user-attachments/assets/a6461cdb-010a-40b4-bd0f-0f0f00d52a86">

@jsandoval81